upgrades.js

```diff
@@ -1,3 +1,5 @@
+/// <reference path="./index.d.ts" />
```

Comment:
Still need this?

Reply:
Yes, it provides type hints and connects the type declaration file to the upgrades file.